planning/autoware_behavior_path_start_planner_module/src/shift_pull_out.cpp

```diff
@@ -324,7 +324,7 @@ std::vector<PullOutPath> ShiftPullOut::calcPullOutPaths(
     shift_line.end = *shift_end_pose_ptr;
     shift_line.end_shift_length = shift_length;
     path_shifter.addShiftLine(shift_line);
-    path_shifter.setVelocity(0.0);  // initial velocity is 0
+    path_shifter.setVelocity(1.0);  // initial velocity is 0
```

Comment:
```suggestion
    path_shifter.setVelocity(1.0);  // initial velocity is 1.0
```

Reply:
could you please add comment?

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/7401/commits/adb799ea0e400b140e84ae0c125004bda3007f91

Reply:
@kosuke55 
added the comment in  https://github.com/autowarefoundation/autoware.universe/pull/7401/commits/5230aaf32e96b97fa9f29f2632e1f29f51f1f789
can you confirm?